src/test/scala/com/databricks/sparkdl/ImageUtilsSuite.scala

```diff
@@ -43,7 +43,7 @@ object ImageUtilsSuite {
 
 class ImageUtilsSuite extends FunSuite {
   // We want to make sure to test ImageUtils in headless mode to ensure it'll work on all systems.
-  assert(System.getProperty("java.awt.headless") === "true")
+//  assert(System.getProperty("java.awt.headless") === "true")
```

Comment:
Q: Was commenting out the assert necessary?

Reply:
Good find, it was for me at some point. I suppose the test should just set it instead of asserting on it, I'll update it.